src/hotspot/os/bsd/gc/x/xPhysicalMemoryBacking_bsd.cpp

```diff
@@ -78,7 +78,7 @@ XPhysicalMemoryBacking::XPhysicalMemoryBacking(size_t max_capacity) :
     _initialized(false) {
 
   // Reserve address space for backing memory
-  _base = (uintptr_t)os::reserve_memory(max_capacity);
+  _base = (uintptr_t)os::reserve_memory(max_capacity, false, mtJavaHeap);
```

Comment:
I think this should be using !MemExec and not a raw 'false' argument.

With that said, I really think it would be best if we actually split os::reserve_memory into two distinct functions:
1) os::reserve_memory(max_capacity, mtJavaHeap) // Non-executable memory
2) os::reserve_executable_memory(max_capacity, mtCode) // Executable

Maybe we could think about that in separate RFE.

Reply:
`false/true` constants are not used in executable args.
separate reserve_memory functions can be left for another RFE.

Reply:
The executable argument really is only false in the original, can we keep this from doing any functional changes here and keep that to separate PR:s?

Reply:
I'm not sure I understand. Are you proposing something else than what I proposed that we could do in a separate RFE?